css-conditional-5/font-tech-format-explainer.md

```diff
@@ -0,0 +1,87 @@
+# `font-tech()` and `font-format()` `<supports-feature>` Extensions to CSS `@supports`
+
+## Authors
+
+* Dominik Röttsches, [@drott](drott@chromium.org)
+* Chris Lilley, [@svgeesus](chris@w3.org)
+* Jonathan Kew, [@jfkthame](jfkthame@gmail.com)
+
+## Introduction
+
+`font-tech()` and `font-format()` are extensions to the CSS Conditional Syntax
+which enable declarative syntax to use a different set of style rules depending
+on font format support, as well as programmatic feature detection of
+capabilities of the font stack. The latter covers a lack of functionality in web
+platform APIs, as most JS APIs and DOM functionality can be easily feature
+detected, but the abilities of the font stack could not be queried in that way
+so far.
+
+The proposed syntax was introduced following discussion in [TAG review
+#666](https://github.com/w3ctag/design-reviews/issues/666) on a previous syntax
+for the `src:` descriptor, and subsequent [resolutions in the CSS
+WG](https://github.com/w3c/csswg-drafts/issues/6520#issuecomment-947810568) to
+harmonize functionality between the `src:` descriptor parsing, and CSS
+Conditional behavior with respect to font feature detection. This had been a
+[request of the
+TAG](https://github.com/w3ctag/design-reviews/issues/666#issuecomment-901220221)
+review.
+
```

Comment:
Somewhere in either the Introduction or the Use cases section I think it's worth a few sentences explicitly tying this to a user need.  In other words, explain (or give examples) what benefits this has for end users, or what end user-facing problems it avoids.

Reply:
Yes. Saying that different platforms and different browsers all started with different color font formats, all of which got standardized into OpenType and thus gave web developers a significant headache, would be helpful. Point out, for those unaware, that these all used the same font format and thus format, and MIME type, can't be used to differentiate them.

Reply:
Added a section starting with "From a mime type and file signature point of view..." to explain how font files look the same from the outside but may required different font technologies to be supported to render correctly, with a link to @svgeesus more detailed history breakdown in the `src:` explainer.